ios/AppDelegate.m

```diff
@@ -11,7 +14,12 @@ @implementation AppDelegate
 - (BOOL) application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
   [MendixAppDelegate application:application didFinishLaunchingWithOptions:launchOptions];
   [self setupUI];
-  
+
```

Comment:
This cannot be enabled for all users to it was moved. See more below
https://github.com/mendix/native-template/pull/157/commits/0edbf49e8a1f7531649df8c4da76e5887c1500e6